docs/01-start-here/01-installation-steps.md

```diff
@@ -47,20 +47,22 @@ You need a Mac or Linux PC (Ubuntu).
 
 ## Automatic
 
+1. Make sure you have `git` installed. Run `brew install git` if required.
+
 1. Check out the repository:
 
     ```bash
     $ git clone git@github.com:guardian/frontend.git
     $ cd frontend
     ```
 
-2. Get AWS Credentials using [Janus](https://janus.gutools.co.uk/) for *Frontend* and *CMS fronts* (you will need access to Janus)
+1. Get AWS Credentials using [Janus](https://janus.gutools.co.uk/) for *Frontend* and *CMS fronts* (you will need access to Janus)
 
-3. Make sure you have the [latest version of Java](#jdk)
+1. Make sure you have the [latest version of Java](#jdk)
```

Comment:
Are they all meant to be numbered 1?

Reply:
Markdown applies the proper numbering itself regardless of the numbers. If you number yourself, then when adding or removing lines, you need to change all the numbers manually to keep things tidy, whereas putting `1` (or any number) everywhere means you don't have that problem :)

Reply:
Ah I did not know that!